spec/ParseObject.spec.js

```diff
@@ -268,18 +268,13 @@ describe('Parse.Object testing', () => {
     });
   });
 
-  it_exclude_dbs(['postgres'])("can set null", function(done) {
+  fit("can set null", function(done) {
```

Comment:
Unfortunately the hooks aren't cloned with the repo. So every dev working on parse-server will have to install that manually